build.gradle.kts

```diff
@@ -98,7 +98,7 @@ configurations {
 }
 
 tasks.compileJava {
-	dependsOn("openApiGeneratePayhub","openApiGeneratePdndClient")
+	dependsOn("openApiGeneratePayhub","openApiGeneratePdndClient","openApiGenerateAnprApiC030", "openApiGenerateAnprApiC003")
```

Comment:
I’m not sure whether to use the entire OpenAPI file. We only need to generate the models using the schemas. Compare the components between the two APIs to see if they are the same; otherwise, I’d generate the models from the generic OpenAPI file.